### PR TITLE
Fix path for headers.md

### LINKED ISSUE #1562: Getting a "Page not found 404" error message
Hello,
When I click on the below link, I am getting the "Page not found 404" error message. 
https://aquasecurity.github.io/tracee/dev/install/headers.md

This page is live and available under Installation > Linux Headers

### PR DESCRIPTION
**Pull request by kaitoii11**
Fixes #1562

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -12,9 +12,9 @@ nav:
     - Building Environment: building/environment.md
     - Building Containers: building/containers.md
     - Building non CO-RE: building/nocore-ebpf.md
+    - Linux Headers: building/headers.md
```
Comment (yanivagman) [inline mkdocs.yml:15]: This should be in the Installation part, and not here

Comment (danielpacak) [reply mkdocs.yml:15]: Okey. Reopening #1562 to put it in the right place.

Comment (kaitoii11) [reply mkdocs.yml:15]: I assumed headers.md was moved to building/ directory intentionally in #1351 but I guess I was wrong. 
I can create a new PR to address this issue as well

Comment (danielpacak) [reply mkdocs.yml:15]: Please do so @kaitoii11 And sorry for confusing you.

Comment (yanivagman) [reply mkdocs.yml:15]: Wait, actually I'm not sure we need it in either places anymore as it is already included in the prerequistes part

**Pull request merged**